pkgs/applications/misc/roxterm/default.nix

```diff
@@ -1,54 +1,38 @@
-{ stdenv, fetchurl, docbook_xsl, dbus, dbus-glib, expat
-, gsettings-desktop-schemas, gdk-pixbuf, gtk3, hicolor-icon-theme
-, imagemagick, itstool, librsvg, libtool, libxslt, makeWrapper
-, pkgconfig, python, pythonPackages, vte
-, wrapGAppsHook}:
+{ at-spi2-core, cmake, dbus, dbus-glib, docbook_xsl, epoxy, fetchpatch, fetchurl
+, glib, gtk3, harfbuzz, hicolor-icon-theme, libXdmcp, libXtst, libpthreadstubs
+, libselinux, libsepol, libtasn1, libxkbcommon, libxslt, p11-kit, pcre
+, pkgconfig, stdenv, utillinuxMinimal, vte, wrapGAppsHook, xmlto
+}:
 
-# TODO: Still getting following warning.
-# WARNING **: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files
-# Seems related to this:
-# https://forums.gentoo.org/viewtopic-t-947210-start-0.html
-
-let version = "3.3.2";
-in stdenv.mkDerivation rec {
-  name = "roxterm-${version}";
+stdenv.mkDerivation rec {
+  pname = "roxterm";
+  version = "3.7.5";
 
   src = fetchurl {
```

Comment:
since your using an archive from GitHub, you're better off replacing with fetchFromGitHub

Reply:
Why is this?  Why is it better to use `fetchFromGitHub` when pulling down an archive than `fetchurl`?

Reply:
``fetchFromGitHub`` is using ``fetchzip`` which is useful for automatically generated archives, where the content of the archive doesn't change but the archive itself might. Also it's just a good practice in case of any future changes to it or if anyone needs to use additional features such as submodules etc.

Reply:
> - using fetchzip which is useful for automatically generated archives, where the content of the archive doesn't change but the archive itself might. 
> - it's just a good practice in case of any future changes to it or if anyone needs to use additional features such as submodules etc.

These two points make sense.  Thanks for the explanation.

I've updated to use `fetchFromGitHub` in commit dfee5f0.